### PR TITLE
Create initial config for DDQA

### PR DESCRIPTION
**Pull request by ofek**
### What does this PR do?

This creates the config for our new QA tool

### Motivation

Moving from Trello to Jira

### Additional Notes

- Feel free to commit directly to my branch
- The names of each team should be their human readable form
- The issue type can be found as the `fields.issuetype.name` returned value in the payload from `/rest/api/2/issue/<ISSUE>`

### CODE REVIEW

File: .ddqa/config.toml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+global_config_source = "aHR0cHM6Ly9yYXcuZ2l0aHVidXNlcmNvbnRlbnQuY29tL0RhdGFEb2cvZ2l0aHViLW1ldGFkYXRhL21hc3Rlci9qaXJhLnRvbWw="
+jira_statuses = [
+  "TODO",
+  "Testing",
+  "Done",
+]
```
Comment (guyarb) [inline .ddqa/config.toml:6]: If my board contains more statuses?

Comment (kacper-murzyn) [reply .ddqa/config.toml:6]: I believe statuses mentioned here would actually be 'labels' from the JIRA perspective, as we don't want to force 20+ projects on their workflows and board organisation. @ofek can provide more implementation details.

Comment (ofek) [reply .ddqa/config.toml:6]: Yes that is correct, for now we will only have a workflow consisting of 3 stages

**Comment by paulcacheux** [comment]
Can you explain in a bit more details what exactly this will create ? Do you have examples on the boards/cards generated by this ?

File: .ddqa/config.toml [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+jira_issue_type = "Task"
+github_team = "processes"
+github_labels = ["team/processes"]
+
+[teams."Windows Agent"]
+jira_project = "WA"
+jira_issue_type = "Task"
+github_team = "windows-agent"
+github_labels = ["team/windows-agent"]
+
```
Comment (joelmarcotte) [inline .ddqa/config.toml:109]:
```suggestion

[teams."Database Monitoring"]
jira_project = "DBM"
jira_issue_type = "Task"
github_team = "database-monitoring"
github_labels = ["team/database-monitoring"]

```

**Comment by 0intro** [comment]
We have just added CSPM Agent.

**Review by mfpierre — APPROVED** [review]

**Comment by sgnn7** [comment]
@ofek / @kacper-murzyn What's blocking the merge of this PR? It's been in limbo for about 3 months at this point.

**Comment by ofek** [comment]
Nothing

**Pull request merged**